frontend/src/Wanted/Missing/MissingRow.js

```diff
@@ -2,15 +2,15 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import episodeEntities from 'Album/episodeEntities';
 import EpisodeTitleLink from 'Album/EpisodeTitleLink';
-import EpisodeStatusConnector from 'Album/EpisodeStatusConnector';
```

Comment:
Lets keep the EpisodeStatusConnector. The column is commented out right now until we refactor it to show a album status (ideas on that are welcome, could be % of tracks, etc..).